### DISCUSSION TITLE
"local" dynamic real for DeepVARHierarchical

### DISCUSSION THREAD

**Discussion by Kub4S** [awslabs/gluonts#2918] [Ideas]
Hello,

I have recently used DeepVARHierarchical estimator for a grouped TS forecasting of order volumes, highly appreciating the reconciliation piece. 

In previous models I supplied local dynamic real feature for each TS enhancing the accuracy (order volume requested for future periods indicating potentially lower need for volumes being requested in actual/forecasted month). With global (common) dynamic real logic I could only aggregated my dyn_ral (future requested volumes) which would be then relevant only for a Total level of hierarchy and doesn't make a sense.

Is there any theoretical plan to introduce such functionality in future, did I miss something, or should I use different model combined with reconciliation in such case?

Many thanks for any opinion.

**Comment by abdulfatir** [comment]
cc @rshyamsundar @melopeo
